.github/actions/verify-token/__tests__/inputs.test.ts

```diff
@@ -20,6 +20,19 @@ import { updateSLSAToken } from "../src/inputs";
 import { rawTokenInterface } from "../src/types";
 
 describe("updateSLSAToken", () => {
+  it("no inputs", async () => {
+    const inputs = JSON.parse("{}");
+    const token = createToken(inputs);
+    const content = `
```

Comment:
can you add a unit tests with no workflow_call? I don't think we have one

Reply:
ignore me. We do have one https://github.com/slsa-framework/slsa-github-generator/blob/b25fcf20ac7cc3869ebf1572bc578552a3548d20/.github/actions/verify-token/__tests__/inputs.test.ts#L178-L180